packages/core/src/app/cart/Redeemable.spec.tsx

```diff
@@ -186,6 +210,68 @@ describe('CartSummary Component', () => {
         });
     });
 
+    describe('when coupon code is collapsed When there is applied error without error message', () => {
+        beforeEach(() => {
+            component = mount(
+              <RedeemableTestComponent
+                appliedRedeemableError={appliedErrorWithoutMessage}
+                applyCoupon={applyCoupon}
+                applyGiftCertificate={applyGiftCertificate}
+                clearError={clearError}
+                onRemovedCoupon={onRemovedCoupon}
+                onRemovedGiftCertificate={onRemovedGiftCertificate}
+                shouldCollapseCouponCode={true}
+              />,
+            );
+        });
+
+        it('renders redeemable toggle link', () => {
+            expect(component.find('[data-test="redeemable-label"]')).toHaveLength(1);
+            expect(component.find('.redeemable-entry')).toHaveLength(0);
+        });
+
+        describe('when redeemable is clicked', () => {
```

Comment:
I feel this is a new test group?
```suggestion
});
        describe('when redeemable is clicked', () => {
```

Reply:
I copy paste another test case: describe('when coupon code is collapsed') 's structure.